src/Schema/Events/ImpressionTracking.ts

```diff
@@ -140,3 +140,94 @@ export interface ValidationAddressViewed {
   subject: string
   option: string
 }
+
+/**
+ * User sees banner "Make an offer, connect now" in CMS.
+ *
+ * This schema describes events sent to Segment from [[SendOffersBannerViewed]].
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "sendOffersBannerViewed",
+ *    context_module: "Home",
+ *    context_page_owner_type: "home"
+ *    partner_id: "61bcda16515b038ce5000104"
+ *  }
+ * ```
+ *
+ */
+export interface SendOffersBannerViewed {
+  action: ActionType.sendOffersBannerViewed
+  context_module: string
+  context_page_owner_type: PageOwnerType
+  partner_id: string
```

Comment:
This modal will be displayed to each browser that accesses the CMS, so it is expected that we have multiple `SendOffersBannerViewed` events for the same `partner_id`. Should we also track a `user_id` as well?

Reply:
Should I add user_id for all events? I thought since they are logged-in, user_id will be by default added.

Reply:
Oh! If that's already available, then please ignore this comment.